packetbeat/_meta/sample_outputs/sip.json

```diff
@@ -0,0 +1,64 @@
+{
+    "@metadata.beat": "packetbeat",
+    "@metadata.type": "_doc",
+    "client.ip": "192.168.1.2",
+    "client.port": 5060,
+    "destination.ip": "212.242.33.35",
+    "destination.port": 5060,
+    "event.action": "sip_register",
+    "event.category": [
+        "network",
+        "protocol",
+        "authorization"
```

Comment:
should be "authentication" = any chance the user.name could be parsed from the auth headers?